torch/export/_trace.py

```diff
@@ -102,6 +102,21 @@ def convert_to_fake(x):
     return fake_args, fake_kwargs, fake_params_buffers, fake_mode
 
 
+def _transform_callable_to_nn_module(f: Callable) -> torch.nn.Module:
```

Comment:
Could you use this utility to convert all the test cases in test_export.py to take nn.module as argument instead?